proxy/logging/LogField.cc

```diff
@@ -164,14 +165,14 @@ namespace
 {
 struct cmp_str {
   bool
-  operator()(ts::ConstBuffer a, ts::ConstBuffer b) const
+  operator()(std::string_view a, std::string_view b) const
   {
-    return ptr_len_casecmp(a._ptr, a._size, b._ptr, b._size) < 0;
+    return ptr_len_casecmp(a.data(), a.size(), b.data(), b.size()) < 0;
```

Comment:
`strcasecmp` is overloaded for `string_view` and `TextView`, so don't use `ptr_len_casecmp`. E.g.
```
return strcasecmp(a, b) == 0;
```